<!-- gps.launch -->
<!--
    This file launches the GPS driver node with
    the appropriate settings defined for our
    GPS model.
    -->
<launch>
    <node name="nmea_serial_driver" pkg="nmea_navsat_driver" type="nmea_serial_driver">
        <param name="port" type="str" value="/dev/igvc_gps" />
        <param name="baud" type="int" value="19200" />
    </node>

    <node name="gps_conv" pkg="gps_common" type="utm_odometry_node">
      <remap from="odom" to="gps_odom"/>
      <param name="rot_covariance" value="99999" />
      <param name="frame_id" value="gps" />
    </node>

</launch>
